packages/now-cli/src/commands/domains/inspect.ts

```diff
@@ -208,8 +159,116 @@ export default async function inspect(
         .join('\n')
     );
 
-    output.print('\n\n');
+    output.print('\n');
+  }
+
+  if (domainConfig.misconfigured) {
+    output.warn(
+      `This domain is not configured properly. To configure it you should either:`,
+      null,
+      null,
+      null,
+      {
+        boxen: {
+          margin: {
+            left: 2,
+            right: 0,
+            bottom: 0,
+            top: 0,
+          },
+        },
+      }
+    );
+    output.print(
+      `  ${chalk.grey('a)')} ` +
+        `Set the following record on your DNS provider to continue: ` +
+        `${code(`A ${domainName} 76.76.21.21`)} ` +
+        `${chalk.grey('[recommended]')}\n`
+    );
+    output.print(
+      `  ${chalk.grey('b)')} ` +
+        `Change your Domains's nameservers to the intended set detailed above.\n\n`
+    );
+    output.print(
+      `  We will run a verification for you and you will receive an email upon completion.\n`
+    );
+
+    const contextNameConst = contextName;
+    const projectNames = Array.from(
+      new Set(projects.map(project => project.name))
+    );
+
+    if (projectNames.length) {
+      projectNames.forEach((name, index) => {
+        const prefix = index === 0 ? '  Read more:' : ' '.repeat(12);
+        output.print(
+          `${prefix} https://vercel.com/${contextNameConst}/${name}/settings/domains\n`
+        );
+      });
+    } else {
+      output.print(`  Read more: https://vercel.link/domain-configuration\n`);
+    }
+
+    output.print('\n');
   }
 
   return null;
 }
+
+async function fetchInformation({
+  output,
+  client,
+  contextName,
+  domainName,
+  cancelWait,
+}: {
+  output: Output;
+  client: Client;
+  contextName: string;
+  domainName: string;
+  cancelWait: () => void;
+}) {
+  const [domain, renewalPrice] = await Promise.all([
+    getDomainByName(client, contextName, domainName, { ignoreWait: true }),
+    getDomainPrice(client, domainName, 'renewal')
+      .then(res => (res instanceof Error ? null : res.price))
+      .catch(() => null),
+  ]);
+
+  if (!domain || domain instanceof DomainNotFound) {
+    cancelWait();
+    output.error(
+      `Domain not found by "${domainName}" under ${chalk.bold(contextName)}`
+    );
```

Comment:
Can we update `DomainNotFound` to include the `contextName` in the error message? Then we can use prettyError() here too?

Reply:
I don't think that's possible in this case. Because we have to style the message that we print with `chalk.bold`.

Reply:
We can't use `chalk.bold` in the error class?

Reply:
I guess we can, but we're not doing it for any other error it seems.
Not sure if having an error message created with chalk makes any difference though.

Reply:
Done

Reply:
Do a search for `under ${context}` in `errors-ts.ts` and there are many messages that print the context but not in bold:

https://github.com/vercel/vercel/blob/421be5d7386c6c7b4da9cf2daf5fa87738b55c0c/packages/now-cli/src/util/errors-ts.ts#L120

Let's decide if bold is necessary (probably a @leo decision) and either use it for all of these errors or none of these errors.

Reply:
This is how it loosk at the moment:

![image](https://user-images.githubusercontent.com/6170607/89290167-bc218180-d658-11ea-853c-37330b4c4174.png)

I think the domain needs to be bold too.